synthesizer/src/vm/mod.rs

```diff
@@ -125,6 +123,23 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
         // Retrieve the block store.
         let block_store = store.block_store();
 
+        #[cfg(not(any(test, feature = "test")))]
+        let mut process = {
+            // Determine the latest block height.
+            let latest_block_height = block_store.current_block_height();
+            // Determine the consensus version.
+            let consensus_version = N::CONSENSUS_VERSION(latest_block_height)?; // TODO (raychu86): Record Commitment - Select the proper consensus version.
+            // Initialize a new process based on the consensus version.
+            if (ConsensusVersion::V1..=ConsensusVersion::V7).contains(&consensus_version) {
+                Process::load_v0()?
```

Comment:
I'll just note that it seems a bit odd that we make process loading during startup dependent on the current consensus version. Are we also updating whenever we reach that consensus version as we're progressing at tip?